src/app/components/datatable-badges.component.ts

```diff
@@ -39,7 +39,7 @@ import { BadgeClass } from '../issuer/models/badgeclass.model';
                         width="40"
                     />
                 </hlm-th>
-                <hlm-th class="!tw-flex-1 tw-justify-center !tw-text-oebblack"><p class="u-text"><time [date]="badge.createdAt" format="dd.mm.yyyy"></time></p></hlm-th>
+                <hlm-th class="!tw-flex-1 tw-justify-center !tw-text-oebblack"><p class="u-text"><time [date]="badge.createdAt" format="dd/MM/y"></time></p></hlm-th>
```

Comment:
Why slash instead of dots? Shouldn't it work just as well with `"dd.MM.y"`?

Reply:
Dots might work too, but we are using Slashes in other places as well and i guess we should stick to one option?

Reply:
yeah stick to "." in german i would say. (we can also add english format for later)

Reply:
changed the format to ```dd.MM.y``` in places where slashes were used. I realized that in the codebase the ```time``` component is mostly used with ```format=mediumDate``` though, which would result in this format: **Apr 15, 2024**. Do you think we should use this format everywhere?